front/app/containers/UsersShowPage/UserEvents.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
```

Comment:
I've added another tab in the user profile, but since you fixed the responsiveness issue in your following i1 work, I'll just merge in your work once it's released :)